autoload/SpaceVim/layers/core/banner.vim

```diff
@@ -390,6 +390,44 @@ function! SpaceVim#layers#core#banner#config() abort
+        \ '=================| |===================================',
+        \ '=========| |===| | | ||_|| |==|  ___)====| |==| || || |',
+        \ '========= \ \=/ /| | |===| |==| |========| |==| || || |',
+        \ '========== \___/=|_|_|===|_|==|_|========|_|===\_____/ ',
+        \ '',
+        \ '               version : ' . g:spacevim_version . '   by : spacevim.org',
+        \ '',
+        \ ],
+        \ [
+        \ '',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMN0kkkkkkkkkkkkkkdl.dKWKkkkkkkkkkkkkkK',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMWo;looooooooool,;;;;;;;;looooooooooool',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMNc;lk0KKKKKKOkl;..;;;:;.lkO0KKKKKK0Oc.',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMM0o,:OKKKKK0l:.;;;;;;;,..oO00KKK0Oo;,d',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:OKKKKK0l;.;;;;:;..ckK00KKKOd;,oKM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:OKKKKK0l;.;:;;..ckKKKKK0Od;,oKMMM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWl:OKKKKK0l;.;..cdkK0KKK0Od;.,lKWMMM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMWO,;OKKKKK0l;..ckKKK0KKK0d;..,::cdKWM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIM0d,.:0KKKKKKxclkKK0KKKK0x:..,;;;;::cdK',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIM....;0KKKKK0kkK0KKKK0ko:..,;;;;;;;;;,,',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMd,:.;0KKKKK000KKKKOkll;..;;;;;;;;;;.,d',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMWKo.;OKK0KKKKKKK0x;................:kX',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWl:OKKKKK0KK0k:..:ll;..col.coo:;lxoo',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:OKKKKKK0kc;..l00o..:Ox:.d0kdodKxo',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:OKKKK0kc;...:OXx;.,xk; .xkoxodOlk',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMWo:0KK0kl;....,d0Oc..lOc..:Ol:llko:O',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMd,lolc,cdo,..........cdkkxkxxxdkxxX',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMNkooooONMWKo,......;xXMMMMMMMMMMMMM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMMMMMMMMMMMMWKo,..;dXMMMMMMMMMMMMMMM',
+        \ 'VIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMVIMMMMMMMMMMMMMMMMMWO.dKMMMMMMMMMMMMMMMMM',
```

Comment:
I think maybe it is better to move logo to the center, if the screen is too narrow, we can not see the logo.

Reply:
@wsdjeg agreed thanks. here is another attempt with more respect to smaller resolutions